SYNTAX.md

```diff
@@ -30,13 +30,27 @@ These are sections that are specified directly within the `pipeline` argument cl
         * *Type*: bareword
         * *Description*: If given, node/image management will need to be specified explicitly in stages and no 
         automatic `checkout scm` call will occur.
-* *Takes a Closure*: No
+* *Takes a Closure*: yes
 * *Examples*:
-    * `agent label:'has-docker', docker:'ubuntu:lts'`
-    * `agent docker:'ubuntu:lts'`
-    * `agent label:'hi-speed'`
-    * `agent none`
-    * `agent any`
+
+```groovy
+agent {
+    label 'has-docker'
+    docker 'ubuntu:lts'
+}
+
+agent {
+    docker 'ubuntu:lts'
+}
+ 
+agent {
```

Comment:
I prefer the original proposal, also... this would make sense to be: `agent any { label: 'hi-speed' }`